build.cmd

```diff
@@ -18,6 +18,12 @@ if %errorlevel% equ 0 (
   set outloop=true
 )
 
+echo %* | findstr /i /C:"FloatingTestRuntimeDependencies=true"  1>nul
+if %errorlevel% equ 0 (
+  set TestFixedRuntimeProjectJson=%~dp0src\Common\test-runtime\LatestDependencies\project.json
+  set defaultBuildConfig=/p:Configuration=Windows_NT_Debug
```

Comment:
E2E App are built from VS. build.cmd is NOT used. E2E App tests the dependencies external customers are going to get. Typically the E2E App will get the dependencies from a pre-release testing feed, such as https://www.myget.org/F/aspnetrc1/api/v2